connector/saml/saml.go

```diff
@@ -408,6 +415,30 @@ func (p *provider) HandlePOST(s connector.Scopes, samlResponse, inResponseTo str
 		}
 		ident.Groups = groups
 	}
+
+	if len(p.allowedGroups) == 0 {
+		// No allowed groups set, just return the ident
+		return ident, nil
+	}
+
+	// Look for membership in one of the allowed groups
+	allowed := false
```

Comment:
Can we use the `groups.Filter` utility method here?

Reply:
I hadn't seen that method, will investigate.

Reply:
I've updated to use `groups.Filter`.